webapp/src/App.js

```diff
@@ -14,34 +14,36 @@ class App extends Component {
     return (
       <div>
         <div className="row">
-          <h2><img src={logo} className="App-logo" alt="logo" />Welcome to go-dart</h2>
-        </div>
-        <div className="row">
-          <div className="col s12 m4">
+
+          <div className="center">          
+            <img  src={logo} className="App-logo" alt="logo" />
+          </div>
+          <h4 className="center header-block">Welcome to godart</h4>
+
+          <div className="col s12 m4" >
             <div className="icon-block">
-              <h2 className="center light-blue-text"><i className="material-icons">add_circle</i></h2>
               <div className="center">
-                <Link to="/newGame" className="btn-large waves-effect waves-light orange">Get started</Link>
+                <Link to="/newGame" className="btn-large waves-effect waves-light btn"><i className="material-icons left large ">add</i>New Game</Link>
```

Comment:
It should be light-blue or all the light-blue elements should be green as this one. But we shouldn't mix colors